website/docs/r/lightsail_container_service.html.markdown

```diff
@@ -0,0 +1,213 @@
+  name = "container-service-1"
+  power = "nano"
+  scale = 1
+  is_disabled = false
+
+  # deployment {
+    # example below
+  # }
+
+  # public_domain_names {
+    # example below
+  # }
+
+  tags = {
+    foo1 = "bar1"
+    foo2 = ""
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name for the container service. Names must be of length 1 to 63, and be
+  unique within each AWS Region in your Lightsail account.
+* `power` - (Required) The power specification for the container service. The power specifies the amount of memory,
+  the number of vCPUs, and the monthly price of each node of the container service.
+* `scale` - (Required) The scale specification for the container service. The scale specifies the allocated compute
+  nodes of the container service.
+* `is_disabled` - (Optional) A Boolean value indicating whether the container service is disabled. Defaults to `false`.
+* `deployment` - (Optional) A deployment specifies the containers that will be launched on the container service and
+  their settings, such as the ports to open, the environment variables to apply, and the launch command to run. It also
+  specifies the container that will serve as the public endpoint of the deployment and its settings, such as the HTTP or
+  HTTPS port to use, and the health check configuration. Defined below.
+* `public_domain_names` - (Optional) The public domain names to use with the container service, such as example.com
+  and www.example.com. You can specify up to four public domain names for a container service. The domain names that you
+  specify are used when you create a deployment with a container configured as the public endpoint of your container
+  service. If you don't specify public domain names, then you can use the default domain of the container service.
+  Defined below.
+    * WARNING: You must create and validate an SSL/TLS certificate before you can use public domain names with your
+      container service.
+      For more information, see
+      [Enabling and managing custom domains for your Amazon Lightsail container services](https://lightsail.aws.amazon.com/ls/docs/en_us/articles/amazon-lightsail-creating-container-services-certificates).
+* `tags` - (Optional) Map of container service tags. To tag at launch, specify the tags in the Launch Template. If
+  configured with a provider
+  [`default_tags` configuration block](https://www.terraform.io/docs/providers/aws/index.html#default_tags-configuration-block)
+  present, tags with matching keys will overwrite those defined at the provider-level.
+
+### deployment
+
+#### Example Usage
+
+```terraform
+resource "aws_lightsail_container_service" "my_container_service" {
+  # ... other configuration ...
+
+  deployment {
+    container {
+      container_name = "hello-world"
+      image = "amazon/amazon-lightsail:hello-world"
+
+      command = []
+
+      environment {
```

Comment:
Can you provide a sample command. Maybe just `yum update -y` ?

Reply:
I believe `command` overrides all the commands the docker image contains when the image is built (kind of seldomly used?).